biopet-extensions/src/main/scala/nl/lumc/sasc/biopet/extensions/gatk/MuTect2.scala

```diff
@@ -0,0 +1,233 @@
+package nl.lumc.sasc.biopet.extensions.gatk
+
+import java.io.File
+
+import nl.lumc.sasc.biopet.core.ScatterGatherableFunction
+import nl.lumc.sasc.biopet.utils.config.Configurable
+import org.broadinstitute.gatk.utils.commandline.{Argument, Gather, Input, Output}
+
+class MuTect2(val parent: Configurable) extends CommandLineGATK with ScatterGatherableFunction {
+  scatterClass = classOf[LocusScatterFunction]
+  setupScatterFunction = { case scatter: GATKScatterFunction => scatter.includeUnmapped = false }
+
+  def analysis_type: String = "MuTect2"
+
+  /** Bam file for the tumor sample. */
+  @Input(fullName = "tumor_bam", required = true)
+  var tumorSampleBam: File = _
+
+  /** Bam file for the normal sample. */
+  @Input(fullName = "normal_bam", required = true)
```

Comment:
I don't understand why we should ignore their statements that tumor-only calling is not supported. This is clearly written in the manual for MuTect2 and the Best Practices page for the tool also recommends using it only with tumor-normal pairs.